spec/lib/myfinance/resources/payable_account_spec.rb

```diff
@@ -0,0 +1,85 @@
+require "spec_helper"
+
+describe Myfinance::Resources::PayableAccount do
+  describe "#create", vcr: true do
+    let(:params) { { due_date: '2015-08-15', amount: 150.99 } }
+    subject { client.payable_accounts.create(3798, params) }
+
+    context "with success" do
+      it "creates a new object" do
+        expect(subject.id).to eq(1215631)
+        expect(subject.due_date).to eq(Date.new(2015, 8, 15))
+        expect(subject.entity_id).to eq(3798)
+        expect(subject.status).to eq(1)
+        expect(subject.status_name).to eq('unpaid')
+        expect(subject.occurred_at).to be_nil
+        expect(subject.amount).to eq(150.99)
+        expect(subject.ticket_amount).to be_nil
+        expect(subject.interest_amount).to be_nil
+        expect(subject.discount_amount).to be_nil
+        expect(subject.total_amount).to be_nil
+        expect(subject.description).to be_nil
+        expect(subject.document).to be_nil
+        expect(subject.document_emission_date).to be_nil
+        expect(subject.observation).to be_nil
+        expect(subject.remind).to be_falsy
+        expect(subject.reminded_at).to be_nil
+        expect(subject.income_tax_relevant).to be_falsy
+        expect(subject.category_id).to be_nil
+        expect(subject.classification_center_id).to be_nil
+        expect(subject.expected_deposit_account_id).to be_nil
+        expect(subject.recurrence_id).to be_nil
+        expect(subject.person_id).to be_nil
+        expect(subject.created_at).to eq(DateTime.parse("2015-08-05T10:31:28-03:00"))
+        expect(subject.updated_at).to eq(DateTime.parse("2015-08-05T10:31:28-03:00"))
+        expect(subject.recurrent).to be_falsy
+        expect(subject.parcelled).to be_falsy
+        expect(subject.recurrence_period).to be_nil
+        expect(subject.number_of_parcels).to be_nil
+        expect(subject.current_parcel).to be_nil
+        expect(subject.competency_month).to eq("2015-08")
+        expect(subject.financial_account_taxes_attributes).to be_empty
+      end
+
+      context "when the payable_account has occurred_at value defined" do
+        let(:params) { { due_date: '2015-08-01', amount: 150.99, occurred_at: '2015-08-05', total_amount: 150.99 } }
```

Comment:
Line is too long. [116/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.